### PR TITLE
pyterm: fix logging directory

### PR DESCRIPTION
**Pull request by OlegHahm**
The default logging directory should be ${HOME}/${HOST}/run-name/

### CODE REVIEW

File: dist/tools/pyterm/pyterm [hunk 1]
```diff
@@ -138,7 +138,7 @@ class SerCmd(cmd.Cmd):
 
         # create Logging object
         my_millis = "%.4f" % (time.time())
-        date_str = '%s.%s' % (time.strftime('%Y%m%d-%H:%M:%S'), my_millis[-4:])
+        date_str = '%s.%s' % (time.strftime('%Y%m%d-%H.%M.%S'), my_millis[-4:])
```
Comment (kaspar030) [inline dist/tools/pyterm/pyterm]: unrelated?

Comment (kaspar030) [inline dist/tools/pyterm/pyterm]: seems minor enough, though...

Comment (OlegHahm) [inline dist/tools/pyterm/pyterm]: Yes, it's somewhat unrelated, but the variable was not evaluated before anyway. But AFAIK MS Windows cannot handle colon in file names.

**Comment by cgundogan** [comment]
ACK

**Comment by cgundogan** [comment]
both CIs agree - GO
